.github/workflows/mend.yml

```diff
@@ -37,3 +39,12 @@ jobs:
         WS_USERKEY: ${{ secrets.MEND_TOKEN }}
         WS_PRODUCTNAME: CD4PE
         WS_PROJECTNAME:  ${{ github.event.repository.name }}
+    - name: "report vulnerabilities"
+        id: vulnerabilities
+        uses: puppetlabs/get-mend-vulnerabilities@v2
+        with:
+          product_token: ${{ secrets.MEND_PRODUCT_TOKEN }}
```

Comment:
I added this secret, the same as the one used for other components in the CD4PE Mend product.